apps/src/templates/sectionProgress/progressTables/ProgressTableSummaryCell.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import color from '@cdo/apps/util/color';
+
+const styles = {
+  container: {
+    height: 20,
+    margin: 10,
+    boxSizing: 'border-box',
+    borderWidth: 1,
+    borderStyle: 'solid'
+  }
+};
+
+export default class ProgressTableSummaryCell extends React.Component {
+  static propTypes = {
+    studentId: PropTypes.number.isRequired,
+    statusPercents: PropTypes.object,
+    assessmentStage: PropTypes.bool,
```

Comment:
Do we have (or want to start) a convention for naming boolean props (e.g. `isAssessmentStage`)?  I don't have a super strong opinion on this but figured I'd ask.

Reply:
I like the convention of having `is-` prefixed named booleans, happy to change it here and use this convention moving forward. Does that sound ok, @cforkish?

Reply:
sgtm!